keylime/ca_impl_cfssl.py

```diff
@@ -102,7 +103,7 @@ def mk_cacert():
                    }
                      ]
            }
-    data = json.dumps(csr)
+    data = yaml.dump(csr)
```

Comment:
I think these all have to stay JSON (including below), since cfssl is using JSON for inputs and outputs.  

When I ran this I got a lot of errors when generating the certs, which may be related to some of the verifier issues you're seeing.

Reply:
I will take a look, main thing is if its going to try and serialize bytes. If so someone was recommending simplejson (on pypi that will allow byte handling in json).